### PR TITLE
Implement -moz-box-ordinal-group property

### LINKED ISSUE #16000: Support -moz-box-ordinal-group property
`-moz-box-ordinal-group` should be fairly easy to implement.

I guess we should add a new `Integer` predefined type to `values` mod, so that we can have `z-index` and `column-count` to use it as well.

### PR DESCRIPTION
**Pull request by upsuper**
This PR fixes #16000.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/16006)

### CODE REVIEW

File: components/style/properties/helpers/animated_properties.mako.rs [hunk 1]
```diff
@@ -1832,8 +1831,7 @@ impl Interpolate for TransformList {
     }
 }
 
-/// https://drafts.csswg.org/css-transitions-1/#animtype-color
-impl Interpolate for ColorOrAuto {
+impl<T> Interpolate for Either<T, Auto> where T: Interpolate + Copy {
```
Comment (Manishearth) [inline components/style/properties/helpers/animated_properties.mako.rs]: Can you implement this for `Either<T, U>` instead?

**Review by Manishearth — APPROVED** [review]
r=me with one change

**Comment by upsuper** [comment]
FWIW, it doesn't work at the moment. Some changes in layout component needs to be done first :/

**Comment by upsuper** [comment]
@Manishearth there are probably non-trivial changes, so you may want to review again.

**Review by Manishearth — APPROVED** [review]

**Pull request merged**